include/openssl/opensslv.h

```diff
@@ -42,6 +42,11 @@ extern "C" {
 # define OPENSSL_VERSION_NUMBER  0x10101000L
 # define OPENSSL_VERSION_TEXT    "OpenSSL 1.1.1-dev  xx XXX xxxx"
 
+#define OPENSSL_MAKE_VERSION(maj,min,fix,patch) ((0x10000000)+((maj&0xff)<<20)+((min&0xff)<<12)+((fix&0xff)<<4)+patch)
```

Comment:
Some compilers are picky and require an L, i.e. `0x10000000L`

Reply:
Shall I rebase these all back into the original patch?